### PR TITLE
Add newick utils

### PR DESCRIPTION
**Pull request by Cashalow**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

No lua and guile support because guile is not available on conda forge and I ran with problem with a missing lua library (shared one and version 5.1 needed).

Cheers,

### CODE REVIEW

File: recipes/newick_utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+source:
+  fn: newick-utils-{{ version }}.tar.gz
+  url: http://cegg.unige.ch/pub/newick-utils-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+  - gcc  # [not osx]
+  - llvm # [osx]
+  - libxml2
```
Comment (bgruening) [inline recipes/newick_utils/meta.yaml]: please pin libxml to `libxml2 2.9.*`

File: recipes/newick_utils/meta.yaml [hunk 2]
```diff
@@ -23,6 +23,8 @@ requirements:
    
   run:
   - libgcc # [not osx]
+  - icu
```
Comment (bgruening) [inline recipes/newick_utils/meta.yaml]: can you please pin icu to `icu 58.*`

File: recipes/newick_utils/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  build:
+  - gcc  # [not osx]
+  - llvm # [osx]
+  - libxml2 2.9.*
+   
+  run:
+  - libgcc # [not osx]
+  - icu 58.*
+  - libiconv
+  - backports.lzma
```
Comment (bgruening) [inline recipes/newick_utils/meta.yaml]: Why is this needed only as runtime and not during build-time?

Comment (Cashalow) [inline recipes/newick_utils/meta.yaml]: Sorry it's just included when requiring libxml2 I have cleaned the dependencies.

**Comment by bgruening** [comment]
Cool, thanks - looks good now!
